api/specs.proto

```diff
@@ -77,14 +77,14 @@ message ServiceSpec {
 	// For now, this is a simple struct, but this can include future information
 	// instructing Swarm on how this service should work on the particular
 	// network.
-	message NetworkAttachmentConfig {
+	message DeprecatedNetworkAttachmentConfig {
```

Comment:
No need to deprecate this as long as the field types and numbers match up, which they do below.